src/views/Sale/components/HeaderControl/index.tsx

```diff
@@ -200,7 +200,7 @@ export function HeaderControl({ status, showGraph, toggleGraph, isFixed, sale }:
       >
         <ControlTitle>
           {status === 'closed'
-            ? 'Missed out on the sale? You can get ERT on'
+            ? `Missed out on the sale? You can get ${sale.tokenOut.symbol}  on`
```

Comment:
Note: We should probably include more messages for a range of different scenarios

Reply:
Certainly, longer term vision for Aqua will be proper integration with swapr, but for now not a high priority